_rules/audio-or-video-avoids-automatically-playing-audio-x0paj4.md

```diff
@@ -0,0 +1,319 @@
+
+## Background
+
+The [instruments][instrument] used to pass this rule (if any), must meet all level A Success Criteria in order to fully satisfy [Success Criterion 1.4.2 Audio Control][sc142]. These extra requirements are left out of this rule, and should be tested separately.
+
+### Bibliography
+
+- [Understanding Success Criterion 1.4.2: Audio Control](https://www.w3.org/WAI/WCAG21/Understanding/audio-control.html)
+- [Accessible Multimedia](https://developer.mozilla.org/en-US/docs/Learn/Accessibility/Multimedia)
+- [Failure of Success Criterion 1.4.2 for absence of a way to pause or stop an HTML5 media element that autoplays](https://www.w3.org/WAI/WCAG21/Techniques/failures/F93)
+- [G170: Providing a control near the beginning of the Web page that turns off sounds that play automatically](https://www.w3.org/WAI/WCAG21/Techniques/general/G170)
+
+## Test Cases
+
+### Passed
+
+#### Passed Example 1
+
+This `audio` element has (in its default controls) an [instrument][] to pause, stop, or turn the audio volume off.
+
+```html
+<audio src="/test-assets/moon-audio/moon-speech.mp3" autoplay controls></audio>
+```
+
+#### Passed Example 2
+
+This `video` element has (in its default controls) an [instrument][] to pause it, an [instrument][] to stop it, and an [instrument][] to turn its audio volume off.
+
+```html
+<video autoplay controls>
+	<source src="/test-assets/rabbit-video/video.mp4" type="video/mp4" />
+	<source src="/test-assets/rabbit-video/video.webm" type="video/webm" />
+</video>
+```
+
+#### Passed Example 3
+
+This `video` element autoplays and has an [instrument][] to pause, stop, or turn the audio volume off.
+
+```html
+<head>
+	<style>
+		button {
+			color: #000;
+		}
+		button:hover {
+			cursor: pointer;
+			cursor: pointer;
+			background-color: grey;
+			color: white;
+		}
+	</style>
+</head>
+<body>
+	<div id="video-container">
+		<!-- Video -->
+		<video id="video" autoplay>
+			<source src="/test-assets/rabbit-video/video.mp4" type="video/mp4" />
+			<source src="/test-assets/rabbit-video/video.webm" type="video/webm" />
+		</video>
+		<!-- Video Controls -->
+		<div id="video-controls">
+			<button type="button" id="play-pause" class="play">Play</button>
+			<button type="button" id="mute">Mute</button>
+		</div>
+	</div>
+	<script src="/test-assets/80f0bf/no-autoplay.js"></script>
+</body>
+```
+
+### Failed
+
+#### Failed Example 1
+
+This `audio` element autoplays, lasts for more than 3 seconds, and does not have an [instrument][] to pause, stop, or mute the audio.
+
+```html
+<audio src="/test-assets/moon-audio/moon-speech.mp3" autoplay></audio>
+```
+
+#### Failed Example 2
+
+This `video` element's audio autoplays for longer than 3 seconds, and does not have an [instrument][] to pause, stop, or mute the audio
+
+```html
+<video autoplay>
```

Comment:
Should we add a note to these that this will not autoplay in several major browsers, or would it be sufficient to mention that in accessibility support?

Reply:
I've made the accessibility support note more explicit. Let me know if you feel that the note is enough or that it should be added to all examples with autoplay.

Reply:
Is the "not playing" achieved by ignoring the `autoplay` attribute, or by overwritting it? That is on a browser that doesn't autoplay unmuted videos, does `video.getAttribute("autoplay")` still returns `true` (as set in the HTML), or does it returns `false` (ignoring the attribute)?
In the second case, the Applicability already makes the video inapplicable on such browsers… (still need an AS note, but likely not one on each example).

Reply:
Chrome autoplays, Firefox and Safari don't. 
All of them return the empty string, since `autoplay` does not really have a value. Even if it is a boolean attribute, `autoplay='false'` [does not work](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/video#attr-autoplay). It is present and it autoplays, or it is not present and it does not autoplay.